python/ql/src/experimental/semmle/python/frameworks/Stdlib.qll

```diff
@@ -9,3 +9,136 @@ private import semmle.python.dataflow.new.TaintTracking
 private import semmle.python.dataflow.new.RemoteFlowSources
 private import experimental.semmle.python.Concepts
 private import semmle.python.ApiGraphs
+
+private module NoSQL {
+  /** Gets a reference to a `MongoClient` DB. */
+  private API::Node mongoClientInstance() {
+    result = API::moduleImport("pymongo").getMember("MongoClient").getReturn() or
+    result =
+      API::moduleImport("flask_mongoengine")
+          .getMember("MongoEngine")
+          .getReturn()
+          .getMember("get_db")
+          .getReturn() or
+    result = API::moduleImport("mongoengine").getMember("connect").getReturn() or
+    result = API::moduleImport("flask_pymongo").getMember("PyMongo").getReturn()
```

Comment:
Nitpick: I guess the current formatting is allowed by our auto-formatting rules, but using the style below better :blush: 

```suggestion
    result = API::moduleImport("pymongo").getMember("MongoClient").getReturn()
    or
    result =
      API::moduleImport("flask_mongoengine")
          .getMember("MongoEngine")
          .getReturn()
          .getMember("get_db")
          .getReturn()
    or
    result = API::moduleImport("mongoengine").getMember("connect").getReturn()
    or
    result = API::moduleImport("flask_pymongo").getMember("PyMongo").getReturn()
```